### PR TITLE
Security updates

### LINKED ISSUE #353: Security updates, disable CY tests for now

### PR DESCRIPTION
**Pull request by AndyEPhipps**
Fixes #353

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "@storybook/addon-info": "3",
     "@storybook/addon-knobs": "3",
     "@storybook/react": "^3.0.0",
-    "axios": "0",
+    "axios": ">=0.18.1",
```
Comment (Heleen-cr) [inline package.json]: what does `>=` do other than `^` does? Don't you need to specify a range when using `>=` ?

Comment (Heleen-cr) [reply package.json]: oh hang on something to do with only using stable versions?

Comment (AndyEPhipps) [reply package.json]: Just using the suggest range from GH: https://github.com/comicrelief/storybook/network/alerts

Comment (Heleen-cr) [reply package.json]: hm ok can't access that. But i guess it's fine.

**Pull request merged**